fw/makefile

```diff
@@ -5,11 +5,11 @@ F_CPU = 16000000
 F_USB = $(F_CPU)
 OPTIMIZATION = 2
 TARGET = beef
-BEEF_LED_REFRESH ?= 40
+BEEF_LED_REFRESH ?= 240
```

Comment:
reason for this change?

Reply:
We have the headroom and I was getting tired of having to set a higher refresh rate, so I increased the default value.